### ISSUE TITLE
无法load MSRA_NER_ALBERT_BASE_ZH

### ISSUE THREAD

**Issue by wujiekd** [hankcs/HanLP#1954]
**Describe the bug**
无法load模型，我已经下载到本地了，依然不load

**Code to reproduce the issue**

```python
import hanlp
HanLP = hanlp.pipeline() \
    .append(hanlp.load('/home/lukeda/.hanlp/ner/msra_ner_albert_base_20211228_173323'), output_key='ner') 
```

**Describe the current behavior**
执行下载
`Failed to load /home/lukeda/.hanlp/ner/msra_ner_albert_base_20211228_173323
If the problem persists, please submit an issue to https://github.com/hankcs/HanLP/issues/new?labels=bug&template=bug_report.md
When reporting an issue, make sure to paste the FULL ERROR LOG below.
================================ERROR LOG BEGINS================================
OS: Linux-5.4.0-148-generic-x86_64-with-glibc2.17
Python: 3.8.20
PyTorch: 2.1.2+cu121
TensorFlow: 2.13.1
HanLP: 2.1.1`

**Expected behavior**
load这个模型成功

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Linux-5.4.0-148-generic-x86_64-with-glibc2.17
- Python version:3.8.20
- HanLP version:2.1.1

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

* [x] I've completed this form and searched the web for solutions.

**Comment by wujiekd** [comment]
感觉是不是重复下载，本地缓存没有用
